V2/Dockerfile

```diff
@@ -12,6 +12,12 @@ RUN chmod -R 777 /usr /etc /run /var; \
     rm -f /etc/NAE/AppDef.json; \
     echo 'https://%PUBLICADDR%:5902/owncloud/index.php/login?user=%NIMBIXUSER%&password=%RANDOM64%' > /etc/NAE/url.txt
 
+FROM scratch
+
+COPY --from=V1 / /
+
 COPY owncloud-start.sh /usr/local/bin/owncloud-start.sh
 COPY AppDef.json /etc/NAE/AppDef.json
+
+RUN mkdir -p /etc/NAE && touch /etc/NAE/{url.txt,AppDef.json}
```

Comment:
that's not sufficient to speed up the pull - the syntax is as follows:
`RUN mkdir -p /etc/NAE && touch /etc/NAE/screenshot.png /etc/NAE/screenshot.txt /etc/NAE/license.txt /etc/NAE/AppDef.json
`
re: https://jarvice.readthedocs.io/en/latest/docker/#best-practices

Reply:
Ok, I was not sure if I could do that if not these files were present. Fixing.